misc/bazel/cargo-gazelle/src/bin/main.rs

```diff
@@ -49,16 +49,19 @@ fn main() -> Result<(), anyhow::Error> {
 
     for package in packages {
         let crate_config = CrateConfig::new(&package);
-        let additive_content = crate_config.additive_content();
+        tracing::debug!(?crate_config, "found config");
```

Comment:
Was this meant for local debugging or did you mean to leave it in?

Reply:
I meant to leave it in. I tried to sprinkle `debug!` logs throughout the tool so if it's not generating the `BUILD.bazel` file you expect you can more easily introspect what it's doing